sql/core/src/test/scala/org/apache/spark/sql/connector/SimpleWritableDataSource.scala

```diff
@@ -68,7 +63,7 @@ class SimpleWritableDataSource extends SimpleTableProvider with SessionConfigSup
       new CSVReaderFactory(serializableConf)
     }
 
-    override def readSchema(): StructType = tableSchema
+    override def readSchema(): StructType = TestingV2Source.schema
```

Comment:
`SimpleScanBuilder.readSchema` is already `TestingV2Source.schema`, we don't need to override it here.

Reply:
yes, it‘s unnecessary to override the function.